src/file_search.py

```
@@ -1,9 +1,38 @@
-from typing import List
+from typing import List, Optional
 import os
 import sys
 from .pattern_matcher import match_pattern
 
 
+def _format_line_output(
+    line_text: str,
+    line_number: int,
+    filename: Optional[str] = None,
+    show_filename: bool = False,
```

Ruff (0.14.1)

Boolean-typed positional argument in function definition

(FBT001)